pkg/security/tests/module_tester.go

```diff
@@ -878,12 +878,7 @@ func genTestConfigs(cfgDir string, opts testOpts) (*emconfig.Config, *secconfig.
 		return nil, nil, fmt.Errorf("unable to set up datadog.yaml configuration: %s", err)
 	}
 
-	spconfig, err := spconfig.New(sysprobeConfigName)
-	if err != nil {
-		return nil, nil, fmt.Errorf("failed to load config: %w", err)
-	}
```

Comment:
We for sure need to keep this. `spconfig.New` touches global vars, so maybe we can just call it and not store the result anywhere